File: RoomMate.Domain/Dto/HouseWorkDto.cs
using RoomMate.Database.Models.Enums;
using System;

namespace RoomMate.Domain.Dto
{
    public class HouseWorkDto : BaseDto
    {
        public int FlatId { get; set; }

        public DateTime HouseWorkDate { get; set; }

        public string Description { get; set; }

        public int UserId { get; set; }

        public string Username { get; set; }

        public WorkTypeEnum WorkType { get; set; }

        public double? Prices { get; set; }

        public string Login { get; set; }


    }
}

File: RoomMate.Front/RoomMatePortal/src/app/housework/dto/housework-dto.ts
import { WorkTypeEnum } from './work-type-enum.enum';

export class HouseworkDto {
  public id: number;
  public flatId: number;
  public houseWorkDate: Date;
  public description: string;
  public userId: number;
  public username: string;
  public prices: number;
  public workType: WorkTypeEnum = WorkTypeEnum.None;
  public login: string;
}
